src/main/resources/application.yml

```diff
@@ -135,3 +135,12 @@ core:
   protocol: ${IEXEC_CORE_PROTOCOL:http}
   host: ${IEXEC_CORE_HOST:localhost}
   port: ${IEXEC_CORE_PORT:13000}
+
+scone:
```

Comment:
Would it be possible to move this before the tee section to gather properties by kind and keep things organized ?